src/main/resources/global/91-init-share.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# we should get env ${CATALINA_HOME} from upstream container docker
```

Comment:
If we "should" get it, can we add a check for it to be present here, and exit if it is not defined?